resources/views/auth/resetPassword.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('layouts.a')
+@section('content')
+<section class="vh-100" style="background-color: #eee;">
+    <div class="container h-100">
+        <div class="row d-flex justify-content-center align-items-center h-100">
+            <div class="col-lg-12 col-xl-11">
+                <div class="card text-black" style="border-radius: 25px;">
+                    <div class="card-body p-md-5">
+                        <div class="row justify-content-center">
+                            <div class="col-md-10 col-lg-6 col-xl-5 order-2 order-lg-1">
+
+                                <p class="text-center h1 fw-bold mb-5 mx-1 mx-md-4 mt-4">Reset Password</p>
+                                @if ($errors->any())
+                                <div class="alert alert-danger">
+                                    <ul>
+                                        @foreach ($errors->all() as $error)
+                                        <li>{{ $error }}</li>
+                                        @endforeach
+                                    </ul>
+                                </div>
+                                @endif
+                                <form method="POST" action="{{route ('auth.resetPassword')}}" class="mx-1 mx-md-4">
+                                    @csrf
+
+
+                                    <input type="hidden" value="{{ $token }}" name="token" id="form3Example3c" class="form-control" required autofocus />
+
+
+                                    <div class="d-flex flex-row align-items-center mb-4">
+                                        <i class="fas fa-envelope fa-lg me-3 fa-fw"></i>
+                                        <div class="form-outline flex-fill mb-0">
+                                            <input type="email" name="email" id="form3Example3c" class="form-control" required autofocus />
```

Comment:
why people need to fill again their email after being redirect from reset password link ?
we can extract email base on token